src/pybind/mgr/dashboard/controllers/cephfs.py

```diff
@@ -57,7 +57,8 @@ def mds_counters(self, fs_id):
             "mds.imported_inodes",
             "mds.inodes",
             "mds.caps",
-            "mds.subtrees"
+            "mds.subtrees",
+            "mds_mem.ino"
```

Comment:
Should ``mds_mem.dn`` be added here, too?

Reply:
Right now 2 counters are used by frontend in mds chart
- `mds.inodes`
- `mds_server.handle_client_request`

`mds_mem.ino` is added in this change to make consistency with the Ranks table (I assume that originally the author of chart tried to display inodes counter rather than dentries).
Please see the screenshot below:
![screencapture-localhost-4200-2019-06-19-11_08_40](https://user-images.githubusercontent.com/1691518/59734703-0ee97f80-9285-11e9-8a95-b54926a072d7.png)

But eventually this chart might be replaced by a Grafana panel, it's much flexible: we don't need to modify backend/frontend codes when different metrics are needed.

Reply:
> But eventually this chart might be replaced by a Grafana panel, it's much flexible: we don't need to modify backend/frontend codes when different metrics are needed.

Would you mind creating a tracker issue about this suggestion? Thanks!

Reply:
> > But eventually this chart might be replaced by a Grafana panel, it's much flexible: we don't need to modify backend/frontend codes when different metrics are needed.
> 
> Would you mind creating a tracker issue about this suggestion? Thanks!

Sure, please see
http://tracker.ceph.com/issues/40556